src/components/Tabs/Tabs.scss

```diff
@@ -327,24 +326,88 @@ $focus-state-box-shadow-color: rgba(0, 0, 0, 0.8);
 
   .Tab {
     padding: spacing(tight) spacing(extra-tight);
-
-    &:active .Title::before {
-      background: var(--p-action-primary-pressed);
+    &:hover .Title {
+      @media (-ms-high-contrast: active) {
+        outline: rem(1px) solid ms-high-contrast-color('text');
+        &::before {
+          display: none;
+        }
+      }
     }
 
     &:focus .Title::before {
       background: var(--p-border-hovered);
     }
+
+    &:active .Title::before {
+      background: var(--p-surface-primary-selected-pressed);
+    }
   }
 
   .DisclosureActivator {
-    margin: 1px 0 -1px;
-    padding: spacing(tight) spacing(tight) spacing(tight) 0;
+    &:hover .Title {
+      background: none;
+
+      &::before {
+        background: var(--p-border-hovered);
+      }
+
+      @media (-ms-high-contrast: active) {
+        outline: rem(1px) solid ms-high-contrast-color('text');
+
+        // stylelint-disable-next-line max-nesting-depth
+        &::before {
+          display: none;
+        }
+      }
+    }
+
+    &:focus .Title {
+      @include focus-ring($style: 'focused');
+
+      &::before {
+        background: var(--p-border-hovered);
+      }
+      @media (-ms-high-contrast: active) {
+        outline: rem(1px) solid ms-high-contrast-color('text');
+
+        // stylelint-disable-next-line max-nesting-depth
+        &::before {
+          display: none;
+        }
+      }
+    }
+
+    &:active .Title::before {
+      background: var(--p-surface-primary-selected-pressed);
+    }
   }
 
   .Tab-selected {
-    &:focus .Title::before {
-      background: var(--p-action-primary);
+    &:hover .Title {
+      @media (-ms-high-contrast: active) {
```

Comment:
Since we're repeating this in a few places, we should probably stick it in a mixin similar to `high-contrast-outline` 
https://github.com/Shopify/polaris-react/blob/49a6d0de7057b3579ce9a59608a6eb942a86afe1/src/styles/foundation/_accessibility.scss

Reply:
100%, yes will do this! THANK YOUUU